components/collection/drop/GenerativeLayout.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="unlockable-container">
+    <div class="container is-fluid border-top pt-6">
+      <div class="columns is-desktop">
+        <div class="column is-half-desktop mobile-padding">
+          <UnlockableCollectionInfo
+            :collection-id="collectionId"
+            :description="description" />
+          <hr />
+
+          <CollectionUnlockableTag :collection-id="collectionId" />
+
+          <CollectionDropMintSection
+            v-if="!isMobile"
+            :has-user-minted="hasUserMinted"
+            :is-wallet-connecting="isWalletConnecting"
+            :is-image-fetching="isImageFetching"
+            :is-loading="isLoading"
+            :minimum-funds="minimumFunds"
+            :minimum-funds-description="minimumFundsDescription"
+            :max-count="maxCount"
+            :minted-count="mintedCount"
+            :mint-count-available="mintCountAvailable"
+            :mint-button-disabled="mintButtonDisabled"
+            :mint-button-label="mintButtonLabel"
+            :is-holder-of-target-collection="isHolderOfTargetCollection"
+            :holder-of-collection-id="holderOfCollectionId"
+            @mint="handleSubmitMint" />
+        </div>
+
+        <div class="column pt-5 is-flex is-justify-content-center">
+          <CollectionDropGenerativePreview
+            :content="drop.content"
+            :image="drop.image"
+            @select="handleSelectImage" />
+        </div>
+
+        <CollectionDropMintSection
+          v-if="isMobile"
+          class="column"
+          :has-user-minted="hasUserMinted"
+          :is-wallet-connecting="isWalletConnecting"
+          :is-image-fetching="isImageFetching"
+          :is-loading="isLoading"
+          :minimum-funds="minimumFunds"
+          :minimum-funds-description="minimumFundsDescription"
+          :max-count="maxCount"
+          :minted-count="mintedCount"
+          :mint-count-available="mintCountAvailable"
+          :mint-button-disabled="mintButtonDisabled"
+          :mint-button-label="mintButtonLabel"
+          :is-holder-of-target-collection="isHolderOfTargetCollection"
+          :holder-of-collection-id="holderOfCollectionId"
+          @mint="handleSubmitMint" />
+      </div>
+
+      <CollectionUnlockableItemInfo :collection-id="collectionId" />
+      <div class="my-4">
+        <CarouselTypeLatestMints
+          :collection-id="collectionId"
+          interaction="MINT" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import UnlockableCollectionInfo from '@/components/collection/unlockable/UnlockableCollectionInfo.vue'
+
+withDefaults(
+  defineProps<{
+    collectionId: string
+    description?: string
+    drop: any
```

Comment:
`Prop 'description' requires default value to be set`

`Unexpected any. Specify a different type`

Reply:
fixed